.circleci/config.yml

```diff
@@ -231,23 +231,40 @@ jobs:
 
```

Comment:
Should we replace the shellcheck part of this file with the pre-commit hook?

Reply:
I tried to do this, not confident I got it right though.

Reply:
One small issue: `types: [bash]` actually skips everything (even with `pre-commit run --all shellcheck` nothing gets run). Leaving it as the default `types: [shell]` seems to work. My guess is pre-commit isn't smart enough to parse the shebang and realize we're using bash not sh, and is just going off of file extension.

I've pushed a commit to fix this.